app/[locale]/dashboard/components/import/atas/atas-processor.tsx

```
@@ -0,0 +1,444 @@
+                if (atasMappings[header]) {
+                    const key = atasMappings[header] as keyof Trade;
+                    const cellValue = row[index];
+                    
+                    switch (key) {
+                        case 'quantity':
+                            quantity = Math.abs(parseFloat(String(cellValue)) || 0);
+                            item[key] = quantity;
+                            break;
+                        case 'pnl':
+                            const { pnl, error } = formatPnl(cellValue)
+                            if (error) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)